src/test/java/net/sf/jabref/logic/importer/fileformat/BibtexParserTest.java

```diff
@@ -132,7 +132,8 @@ public void singleFromStringReturnsEmptyIfNoEntryRecognized() throws IOException
 
     @Test
     public void parseRecognizesEntry() throws IOException {
-        ParserResult result = new BibtexParser(importFormatPreferences).parse(new StringReader("@article{test,author={Ed von Test}}"));
+        ParserResult result = new BibtexParser(importFormatPreferences)
```

Comment:
We talked about the `deprecated` issue in the last devcall. Since you weren't there, I give you a short summary. 
The consensus was that we should concentrate more on new features / bug fixing instead of cleaning up the code since the code quality is rather high now (at least in logic + model). But of course there are still some problematic code, for example superfluous methods / strange implementation / things which look like duplicated code but there might be subtle differences. If one works one some PR and notice such a flaw in the code, then one can simple mark the corresponding method as deprecated (and document why one did this). Then the deprecated flag just means "try not to use this method anymore". The hope was that while modifying some code which still uses the deprecated method one has enough context to be able to decide if the corresponding call can be replaced by something else. Thus at some point the usage of the deprecated method dies over time and at some point it can be replaced. So the aim was to reduce the amount of pure refactoring/cleanup PRs and let the code evolve over time in the right direction by smaller changes (with context). This of course doesn't mean that cleanup PRs like this one here don't have value - in contrast. But they are are often harder to review.

Anyway, please clean the tests (ready-for-review removed until then)